src/macappkit.m

```diff
@@ -109,10 +109,10 @@
   (@"NSTouchBarItemIdentifierCandidateList")
 #endif
 
-static void mac_within_gui_and_here (void (^ CF_NOESCAPE) (void),
-				     void (^ CF_NOESCAPE) (void));
-static void mac_within_gui_allowing_inner_lisp (void (^ CF_NOESCAPE) (void));
-static void mac_within_lisp (void (^ CF_NOESCAPE) (void));
+static void mac_within_gui_and_here (void (^) (void),
+                                     void (CF_NOESCAPE ^) (void));
```

Comment:
Here the idea is that `block_here` is executed locally, and so does not escape?

Reply:
Exactly that.